amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,71 @@
+= Amazon SQS Connector 5.8 Additional Configuration Information - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-config-topics.adoc
+
+[node-behavior]
+== Receiving Messages on the Primary Node Only
+
+The default behavior of the Receive Messages source is to receive messages on all nodes when running in cluster mode. 
+
+If you want the app to receive messages on the primary node only:
+
+. In Studio, select the *Receive Messages* source for your flow.
+. On the *Advanced* sub-tab of the *Receive messages* tab, select the *Primary node only* checkbox: 
++
+image::amazon-sqs-primary-node-only.png[Read message configuration Advanced tab with Primary node only option selected]
+
+The multi-node cluster behavior depends on the selected SQS queue type. When using the Standard queue, SQS guarantees that a message is delivered at least once. However, duplicate messages might be introduced in the queue. To avoid duplication of messages in the queue, you can use a FIFO (First-In-First-Out) queue to ensure that a message is delivered only once and that it remains available until a consumer processes and deletes it.
+
+For more information about queue types, see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/standard-queues.html[AWS SQS Stanard Queues] and https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html[AWS SQS FIFO Queues].
+
+== Creating the FIFO Queue
+
+FIFO queues deliver messages in the order in which they are sent.
+
+To create the FIFO queue, use the *Create queue* operation in the connector, and set the *FifoQueue* and *ContentBasedDeduplication* field values to true.
+
+End the *queue name* value with the `.fifo` extension, for example: +
+`MyTestFIFOQueue.fifo`.
+
+While creating the FIFO queues, choose either of the two regions that Amazon supports:
```

Comment:
Aws support now 15 regions for fifo queue. Therefore I am not sure whether here is important to include all the regions. What do you think ? https://aws.amazon.com/about-aws/whats-new/2019/02/amazon-sqs-fifo-qeues-now-available-in-15-aws-regions/